### PR TITLE
fix: Missing track selection on negative d0

### PR DESCRIPTION
**Pull request by paulgessinger**

### CODE REVIEW

**Comment by paulgessinger** [comment]
Reference updated.

**Review by andiwand — APPROVED** [review]

File: Examples/Scripts/Python/full_chain_odd.py [hunk 1]
```diff
@@ -105,7 +105,12 @@
     trackingGeometry,
     field,
     CKFPerformanceConfig(ptMin=1.0 * u.GeV if ttbar_pu200 else 0.0, nMeasurementsMin=6),
-    TrackSelectorRanges(pt=(1.0 * u.GeV, None), absEta=(None, 3.0), removeNeutral=True),
+    TrackSelectorRanges(
+        pt=(1.0 * u.GeV, None),
+        absEta=(-3.0, 3.0),
```
[the remark this answers is not in the document]
Comment (paulgessinger) [reply Examples/Scripts/Python/full_chain_odd.py:110]: I don't actually know why I changed this line anyway...

[the remark this answers is not in the document]
Comment (paulgessinger) [reply Examples/Scripts/Python/full_chain_odd.py:110]:
```suggestion
        absEta=(None, 3.0),
```

**Review by andiwand — APPROVED** [review]

**Comment by paulgessinger** [comment]
Updated references again after last update, @andiwand.

**Review by andiwand — APPROVED** [review]

**Pull request merged**